pkg/util/provider/app/app.go

```diff
@@ -279,6 +279,7 @@ func StartControllers(s *options.MCServer,
 		targetCoreInformerFactory.Storage().V1().VolumeAttachments(),
 		machineSharedInformers.MachineClasses(),
 		machineSharedInformers.Machines(),
+		targetCoreInformerFactory.Core().V1().Pods(),
```

Comment:
Nit: Can you move this a few lines up so that all `InformerFactory`s are together?